umbraco-cms/tutorials/creating-a-custom-dashboard.md

```diff
@@ -660,6 +659,155 @@ angular.module("umbraco").controller("CustomWelcomeDashboardController", functio
+
+![Content Node Card]
+
+First, we need to wrap our unordered list and its content in the ```<uui-card-content-node>``` so it will look like this:
+
+```html
+<uui-card-content-node name="The card">
+    <ul class="unstyled">
+        <li ng-repeat="logEntry in vm.UserLogHistory.items">
+            <i class="{{logEntry.Content.icon}}"></i> <a href="/Umbraco/#/{{logEntry.editUrl}}">{{logEntry.Content.name}} <span ng-if="logEntry.comment">- {{logEntry.comment}}</span></a> - <span class="text-muted">(Edited on: {{logEntry.timestamp  | date:'medium'}})</span>
+        </li>
+    </ul>
+</uui-card-content-node>
+```
+
+Let's make sure that the card shows the name of the content nodes that the editors recently worked with.
+To do this we need to replace the name "The card" in the ```<uui-card-content-node">``` with ```{{logEntry.Content.name}}``` so it will look like this:
+
+```html
+<uui-card-content-node name="{{logEntry.Content.name}}"> 
+```
+
+We also need to move the loop that we added into the **uui-card** as well.
+
+Which then will look like this:
+
+```html
+<uui-card-content-node name="{{logEntry.Content.name}}" ng-repeat="logEntry in vm.UserLogHistory.items">
+```
+
+At this point the code looks like this:
+
+```html
+<uui-card-content-node name="{{logEntry.Content.name}}" ng-repeat="logEntry in vm.UserLogHistory.items">
+    <ul class="unstyled">
+        <li>
+            <i class="{{logEntry.Content.icon}}"></i> <a href="/Umbraco/#/{{logEntry.editUrl}}">{{logEntry.Content.name}} <span ng-if="logEntry.comment">- {{logEntry.comment}}</span></a> - <span class="text-muted">(Edited on: {{logEntry.timestamp | date:'medium'}})</span>
+        </li>
+    </ul>
+</uui-card-content-node>
+```
+
+Now we also want the editors to be able to go directly to the content node, let's move the ```<a href="/Umbraco/#/{{logEntry.editUrl}}">{{logEntry.Content.name}} <span ng-if="logEntry.comment">- {{logEntry.comment}}</span></a>``` line down under the ```<uui-card-content-node>```.
+
+Make sure to add some text to the ```<a>``` tag like *"click here"* or *"See Node"* so that our code should look like this:
+
+```html
+<uui-card-content-node name="{{logEntry.Content.name}}" ng-repeat="logEntry in vm.UserLogHistory.items">
+    <a href="/Umbraco/#/{{logEntry.editUrl}}"><span ng-if="logEntry.comment"> {{logEntry.comment}}</span><span style="font-weight: 700">See Node</span></a>
+    <ul class="unstyled">
+        <li>
+            <i class="{{logEntry.Content.icon}}"></i><span class="text-muted">(Edited on: {{logEntry.timestamp  | date:'medium'}})</span>
+        </li>
+    </ul>
+</uui-card-content-node>
+```
+
+Next up, let's go ahead and update the ```<ul>``` tag with the style from the UI library Card with the following:
+
+```style="list-style: none; padding-inline-start: 0px; margin: 0;```
+
+We can also go ahead and remove the ```<i class="{{logEntry.Content.icon}}"></i>``` from our list as we won't be using the icon as the card has one by default.
```

Comment:
Do we need to add this if we are not really using this? It's a little confusing that we are first adding it and then removing it.

Reply:
Which part are you refering to? On line 738 or line 740? :)

Reply:
Line 740.. where it mentions we can also go ahead and remove ... as we won't be using the icon

Reply:
Ahhh okay, i think it makes sense to have this as the <i> tag is something that has been added in the previous part of the tutorial :)